crc-bundle-info.json.sample

```diff
@@ -45,6 +45,8 @@
       # Disk image used by the node, the 'storage' object will contain more
```

Comment:
> From 4.4 release etcd certs are now created for internal IP instead dns srv record like it used to, in case of the libvirt the internal IP is same as VM ip.

Did the certificate directly refer to the SRV records? My understanding is that the SRV records were used to get the hostname of the etcd server, and then the certificates were valid for that hostname.
Maybe
>  Starting from the 4.4 release, etcd certs are now only valid for etcd IP. In previous releases, the etcd instances were accessed through hostnames so the etcd IP was not that important. Now, crc will need to know for which IP these certificates are valid. In the libvirt case, the internal IP is same as VM ip.